setup.cfg

```diff
@@ -14,6 +14,18 @@ source = staticjinja
 [coverage:html]
 directory = .htmlcov
 
+[coverage:report]
+exclude_lines =
+    pragma: no cover
+    if .*TYPE_CHECKING:
+    if False:
+    def __repr__
+    class .*Protocol
+
```

Comment:
As above, if easy can you put sort these A-Z? Thanks

Reply:
:+1: